sdk/maps/azure-maps-timezone/src/samples/java/com/azure/maps/timezone/samples/ConvertWindowsTimezoneToIana.java

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.maps.timezone.samples;
+
+import com.azure.core.http.policy.HttpLogDetailLevel;
+import com.azure.core.http.policy.HttpLogOptions;
+import com.azure.identity.DefaultAzureCredential;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import com.azure.maps.timezone.TimezoneAsyncClient;
+import com.azure.maps.timezone.TimezoneClient;
+import com.azure.maps.timezone.TimezoneClientBuilder;
+
+public class ConvertWindowsTimezoneToIana {
+    public static void main(String[] args) {
+        TimezoneClientBuilder builder = new TimezoneClientBuilder();
+
+        // Authenticates using subscription key
+        // AzureKeyCredential keyCredential = new AzureKeyCredential(System.getenv("SUBSCRIPTION_KEY"));
+        // builder.credential(keyCredential);
+
+        // Authenticates using Azure AD building a default credential
+        // This will look for AZURE_CLIENT_ID, AZURE_TENANT_ID, and AZURE_CLIENT_SECRET env variables
+        DefaultAzureCredential tokenCredential = new DefaultAzureCredentialBuilder().build();
+
+        builder.credential(tokenCredential);
+        builder.timezoneClientId(System.getenv("MAPS_CLIENT_ID"));
+        builder.httpLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BODY_AND_HEADERS));
+        TimezoneClient client = builder.buildClient();
```

Comment:
nit:, please use fluent style (`new Builder().foo().bar().build()`) - we use it for other SDKs